package.json

```diff
@@ -88,72 +90,60 @@
     "babel-loader": "^8.0.5",
     "babel-plugin-dev-expression": "^0.2.2",
     "babel-plugin-dynamic-import-node": "^2.2.0",
+    "babel-plugin-inline-react-svg": "^1.1.0",
     "babel-plugin-lodash": "^3.3.2",
     "babel-plugin-module-resolver": "^3.2.0",
     "babel-plugin-react-docgen": "^3.0.0",
     "codecov": "^3.2.0",
+    "commitizen": "^3.1.2",
     "component-playground": "^3.2.1",
-    "cross-env": "^5.2.0",
+    "cross-env": "^6.0.3",
+    "css-loader": "^3.2.0",
+    "cz-customizable": "^6.2.0",
+    "cz-customizable-ghooks": "^1.5.0",
     "docz": "^1.2.0",
     "docz-core": "^1.2.0",
     "docz-theme-default": "^1.2.0",
     "eslint-plugin-prettier": "^3.1.0",
     "expect": "^24.5.0",
     "gh-pages": "^2.0.1",
     "glob": "^7.1.3",
-    "husky": "^1.3.1",
+    "husky": "^3.0.8",
     "jest": "^24.5.0",
     "jest-axe": "^3.1.1",
     "jest-cli": "^24.5.0",
     "jest-emotion": "^10.0.10",
     "license-checker": "^25.0.1",
-    "onchange": "^5.2.0",
+    "onchange": "^6.1.0",
     "postcss-loader": "^3.0.0",
     "postcss-object-fit-images": "^1.1.2",
-    "prettier": "1.16.4",
-    "prettier-eslint": "^8.8.2",
+    "prettier": "1.18.2",
+    "prettier-eslint": "^9.0.0",
     "react-docgen": "^4.1.0",
     "react-docgen-annotation-resolver": "^1.1.0",
-    "react-svg-loader": "^2.1.0",
+    "react-svg-loader": "^3.0.3",
     "react-test-renderer": "^16.8.6",
-    "storybook-addon-smart-knobs": "^4.1.0",
-    "style-loader": "^0.23.1",
+    "style-loader": "^1.0.0",
     "webpack-merge": "^4.2.1"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.10",
     "@emotion/is-prop-valid": "^0.8.2",
-    "@emotion/styled": "^10.0.10",
-    "@svgr/webpack": "^4.1.0",
-    "babel-plugin-inline-react-svg": "^1.1.0",
-    "commitizen": "^3.0.5",
-    "css-loader": "^2.1.1",
-    "cz-customizable": "^5.3.0",
-    "cz-customizable-ghooks": "^1.5.0",
-    "dom-helpers": "^3.4.0",
-    "downshift": "^1.31.14",
-    "emotion": "^10.0.9",
-    "emotion-theming": "^10.0.10",
-    "facepaint": "^1.2.0",
-    "hoist-non-react-statics": "^3.3.0",
+    "dom-helpers": "^5.1.0",
+    "downshift": "^3.3.4",
     "keycode": "^2.2.0",
-    "lint-staged": "^8.1.5",
     "lodash": "^4.17.11",
     "markdown-to-jsx": "^6.9.3",
     "moment": "^2.24.0",
     "no-scroll": "^2.1.1",
     "polished": "^3.1.0",
     "prop-types": "^15.7.2",
     "prop-types-exact": "^1.2.0",
-    "react": "^16.8.6",
-    "react-dates": "^16.6.1",
-    "react-dom": "^16.8.6",
```

Comment:
Since npm 3 (and since always in yarn) peer dependencies are not installed automatically. Instead, they should be listed as dev dependencies (this allows you to pin them to a specific version as well.

Reply:
cool, that's what I wanted to do 👍